libs/pages/clusters/src/lib/feature/page-clusters-create-feature/page-clusters-create-feature.tsx

```diff
@@ -38,18 +41,23 @@ export const useClusterContainerCreateContext = () => {
   return clusterContainerCreateContext
 }
 
-export const steps = (cloudProvider: CloudProviderEnum = 'AWS', clusterType?: string) => {
-  return match(cloudProvider)
-    .with('SCW', () => [
+export const steps = (clusterGeneralData?: ClusterGeneralData, clusterType?: string) => {
```

Comment:
We have a little regression here because we need to have an default value to avoid it:
<img width="59" alt="Capture d’écran 2024-01-15 à 17 08 03" src="https://github.com/Qovery/console/assets/533928/8447910d-bb09-4943-ad89-c32610e5ca5c">

We can hide this indicator if `steps` is `undefined` or you can add default value with `AWS` & `MANAGED`

Reply:
https://github.com/Qovery/console/pull/1117/commits/cf7c20bfccf6004ee3a492ebd9154fd8fb3e2d1d